gpytorch/kernels/rff_kernel.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python3
+
+import math
+from typing import Optional
+
+import torch
+from torch import Tensor
+
+from ..lazy import MatmulLazyTensor, RootLazyTensor
+from .kernel import Kernel
+
+
+class RFFKernel(Kernel):
+    r"""
+    Computes a covariance matrix based on Random Fourier Features with the
+    RBFKernel.
+
+    See Random Features for Large-Scale Kernel Machines by Rahimi and Recht
+
+    Here we use sine and cosine features which gives a lower-variance estimator.
+    See On the Error of Random Fourier Features by Sutherland and Schneider.
+
+    Args:
```

Comment:
Could you update the Args and Attributes to use the correct sphinx style (which, I admit, is not what we have in most places in the code)? See ArcKernel for an example.